### ISSUE TITLE
Voiceover support (iOS)

### ISSUE THREAD

**Issue by ghost** [jitsi/jitsi-meet#5852]
Voice over support for blind users would be great I saw a review and it was one downside for him or her.

**Comment by Echolon** [comment]
related:

#4541 **Toggle buttons on the UI not accessible with VoiceOver on Mac OS and Google Chrome**
#3590 **Accessibility issue: unknown state of toggle buttons in web page and android app**
#5817 **Various accessibility issues in the React Native code**

**Comment by Echolon** [comment]
Hi,
The team is aware of accessibility. Any reports are now grouped in the pinned issue #6090.

---
*I'm a community contributor and not directly affiliated with the Jitsi team.*

**Comment by guenterhanke** [comment]
Hi Developers,
i am blind and have problems using the Jitsi App on my iPhone 7 under IOS 14.3 with VoiceOver.
App version is 20.6.2 Build 103

I faced the following accessibility problems with VoiceOver:

1. after i have clicked on a meeting link and clicked start in Jitsi App
- i see the "please enter your name" but VoiceOver does not show any text box to enter my name
- i can't click the buttons eg. mute audio, enable video or leave meeting

So i can not use the app at all


2. when i open the app without clicking a link
- the first element is an unlabelled graphic.
Voiceover tells me "heaven"
It is the menu button.
Please label this important graphic with an accessible text.
- between video and audio there is a "0" standing
what is it for? the state of video or audio and 0=off?


3. in the settings screen
when i have clicked the "heaven" icon
- for VoiceOver the screen stays the same but at the end the settings screen is inserted.
This is bad for blind users, please consider to take a separate dialogue for the settings screen
- all checkbox states are described as "1" or "0", i am not sure if 0 is off or not


Steps to reproduce:
- switch on VoiceOver 
- try to take part via meeting link


However, the browser based version with Chrome under Win 10 is running very good with screen reader.
Thank you for that
Greetings
Guenter

**Comment by l3d00m** [comment]
I guess this is still relevant?
